index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Challenge #1 - Landing Page</title>
+    <link rel="stylesheet" href="style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="theme-color" content="#47D95E">
+</head>
+<body>
+    <div class="container">
+        <header class="header">
+            <img class="header__img" src="assets/undraw_developer_activity_bv83.svg" alt="An illustration of a developer sitting at a desk with a computer">
```

Comment:
Ilustration w alcie jest niepotrzebne już z samego tagu IMG wynika że jest to obrazek, ilustracja